DependencyInjection/ProfilerExtension.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace Http\HttplugBundle\DependencyInjection;
+
+use Http\HttplugBundle\Collector\DebugPlugin;
+use Symfony\Component\Config\FileLocator;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Loader\XmlFileLoader;
+use Symfony\Component\DependencyInjection\Reference;
+
+/**
+ * This class is responsible for loading profiler tools and the toolbar. This extension should generally not be used
+ * in production.
+ *
+ * @author David Buchmann <mail@davidbu.ch>
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+class ProfilerExtension
```

Comment:
I still think it would be better to have all the code in one class, even if it is large. IMO it is easier to read.